src/containers/shared/components/TransactionTable/styles.scss

```diff
@@ -37,7 +38,15 @@
     @include semibold;
 
     @include for-size(desktop-up) {
-      width: 260px;
```

Comment:
The widths should be based on whether or not there is the presence of the new column

Reply:
I see what youre saying but 230px doesnt cut the types, and the code is pretty ugly with logic that assigns the css based off the existence of this column.